pubsub/README.md

```diff
@@ -49,93 +48,96 @@ delivery](https://cloud.google.com/pubsub/overview#benefits).  This means that
+Alternately, you can install the SDK according to [these instructions](https://cloud.google.com/sdk/downloads), and run locally, e.g. on your laptop. If you install locally, make sure that you've authenticated using your Google Account:
+
+```sh
+gcloud auth login
+```
 
-### Install Docker
+then **set the default project** for the Cloud SDK to your project:
 
-This app requires that you have
-[Docker](https://docs.docker.com/installation/#installation) installed locally.  This is because you will build a custom Docker image for your Kubernetes app.  Follow the installation instructions on the docker site.
+```sh
+gcloud config set project <your_project_id>
+```
 
-### Create and configure a new Google Cloud Platform project
+In the Cloud Shell, you won't have to auth, but confirm you're using the correct project via:
 
-To work through this example, you must have a Google Cloud Platform project with
-the required APIs enabled. In the [Cloud Developers Console](https://console.developers.google.com),
-create a new project or choose an existing project, then in the `APIs & auth > APIs` panel,
-enable the BigQuery, Google Compute Engine, PubSub, Google Cloud Storage, and Google Cloud Storage JSON APIs.
-You will be prompted to enable billing if you have not previously done so.
+```sh
+gcloud config list project
+```
 
-### Set up the Google Cloud SDK
+and set it as above if necessary.
 
-This tutorial uses the [Google Cloud SDK](https://cloud.google.com/sdk/) to interact with the Cloud Platform from
-your local terminal. Follow the instructions on that page to install the Cloud SDK.
+### Clone the example repo
 
-**Enable preview features** in the gcloud tool, as follows:
+If you haven't already, clone the Github repository for this example:
 
-    $ gcloud components update preview
+```sh
+git clone https://github.com/GoogleCloudPlatform/kubernetes-bigquery-python.git
+```
 
-and **authenticate** using your Google Account:
+(If you prefer, you can
+[download](https://github.com/GoogleCloudPlatform/kubernetes-bigquery-python/archive/master.zip) it instead.)
 
-    $ gcloud auth login
+Then change to the `pubsub` subdirectory of the repo:
 
-After authorizing, **set the default project** for the Cloud SDK to the
-project you selected in the previous section of this tutorial:
+```sh
+cd kubernetes-bigquery-python/pubsub
+```
 
-    $ gcloud config set project <project_id>
+We'll stay in this directory for the rest of the tutorial.
 
 
 ### Create a BigQuery table
 
 Next, create a BigQuery table to store your tweets. BigQuery groups
-tables into abstraction layers called datasets, so first create a dataset as necessary.  You can do this from the web UI, or from the command line like this:
+tables into abstraction layers called datasets, so first create a dataset as necessary.  You can do this from the [web UI](xxx), or from the command line like this:
```

Comment:
I don't know if we can link directly to the create a table / dataset dialog. There are screenshots at https://cloud.google.com/bigquery/docs/tables#create-table on how to create a table with the web UI, though.

Reply:
oops, I'd missed that dead link too-- thanks.